include/jemalloc/internal/sz.h

```diff
@@ -150,14 +154,13 @@ sz_size2index_compute(size_t size) {
 JEMALLOC_ALWAYS_INLINE szind_t
 sz_size2index_lookup(size_t size) {
 	assert(size <= SC_LOOKUP_MAXCLASS);
-	szind_t ret = (sz_size2index_tab[(size-1) >> SC_LG_TINY_MIN]);
+	szind_t ret = (sz_size2index_tab[(size+7) >> SC_LG_TINY_MIN]);
```

Comment:
How is it generally correct to change from `size-1` to `size+7`?  My memory of how this all works is a bit fuzzy, but on the surface this sure seems like it's assuming the word size is 8.

Reply:
Ah, right, this should be size+SC_LG_TINY_MIN?